db/migrate/20150819140659_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
There is 1 huge disadvantage of this approach and that models can get renamed probably over time, a then your migrations won't run from start till end. Rather than this I would recommend custom rake tasks, which do not depend on schema migrations, but are rather better for data migrations.